server/models/Capture.js

```diff
@@ -22,8 +22,8 @@ const NewCapture = ({ id, reference_id, image_url, lat, lon, planter_id, planter
     lat,
     lon,
     planter_id,
-    planter_contact: planter_identifier, //planter_identifier is legacy terminology replaced by planter_contact and exists for data-pipeline compatibility
-    status: 'unverified',
+    planter_username: planter_identifier, //planter_identifier is legacy terminology, it represents the username of the planter, refactored to planterUsername

```

Comment:
Replacing planter_identifier with planter_username. This avoids the confusion with planter_id  which is different from planter_identifier.